assets/app/view/game/round/auction.rb

```diff
@@ -32,12 +33,14 @@ def render
 
           store(:before_process_pass, -> { hide! }, skip: true) if @current_actions.include?('pass')
 
-          if @current_actions.include?('par')
+          if @current_actions.include?('par') && @step.companies_pending_par
             h(:div, render_company_pending_par)
           else
             h(:div, [
+              render_simple_bid,
               render_show_button,
               *render_companies,
+              *render_corporations,
```

Comment:
i don't think auctions need to show corporations

Reply:
In the initial auction in 1860 you are bidding on the right to choose what private company to buy or corporation to start. It is important to see those choices, including the corporations, when bidding.